modules/gateway-task/main.tf

```diff
@@ -105,7 +105,7 @@ resource "aws_ecs_task_definition" "this" {
               initProcessEnabled = true
             }
             healthCheck = {
-              command  = ["CMD-SHELL", "curl localhost:10000/consul-ecs/health"] # consul-ecs-control-plane exposes a listener on 10000 to indicate it's readiness
+              command  = ["CMD-SHELL", "curl -f localhost:10000/consul-ecs/health || exit 1"] # consul-ecs-control-plane exposes a listener on 10000 to indicate it's readiness
```

Comment:
Isn't  the `|| exit 1` redundant after adding the curl flag `-f`?

Reply:
Took this suggestion in https://github.com/hashicorp/terraform-aws-consul-ecs/pull/243